server/src/routes/index.ts

```diff
@@ -39,7 +39,24 @@ router.get('/api-docs', swaggerUI.setup(openapiSpecification));
 
 /**
  * @swagger
+ * components:
+ *   securitySchemes:
+ *     jwtCookie:
+ *       type: apiKey
+ *       in: cookie
+ *       name: jwt
  * definitions:
+ *   Failure:
+ *     type: object
+ *     description: A reason for a failure, with a chiefly developer-facing error message
+ *     properties:
+ *       success:
+ *         type: boolean
+ *         description: '`false` to indicate failure'
+ *         example: false
+ *       message:
```

Comment:
If `Failure` is representing what gets returned upon failure from any API route, then this should be `errors`, which is an array of all errors which occurred, since Yup may return multiple errors.